cfn-resources/custom-db-role/test/cfn-test-create-inputs.sh

```diff
@@ -11,7 +11,7 @@ set -o pipefail
 set -x
 
 function usage {
-    echo "Creates a new customdb role for the test"
+	echo "Creates a new customdb role for the test"
```

Comment:
`shfmt`?

Reply:
we do use `shfmt` as part of the pre-commit hook https://github.com/mongodb/mongodbatlas-cloudformation-resources/blob/57f6ace81eb39e80ac7ffbf86deee47da84b5509/.githooks/pre-commit#L26

I think this is the reason why I add these changes here